models/omen/gnosis/omen_gnosis_liquidity.sql

```diff
@@ -0,0 +1,140 @@
+{{ config(
+    schema = 'omen_gnosis',
+    alias = 'liquidity',
+    
+    partition_by = ['block_day'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_time', 'tx_hash', 'evt_index'],
+    post_hook='{{ expose_spells(\'["gnosis"]\',
+                                "project",
+                                "omen",
+                                \'["hdser"]\') }}'
```

Comment:
We may use a better format for the post_hook here (and in future):
```suggestion
    post_hook='{{ expose_spells(blockchains  = \'["gnosis"]\',
                                spell_type   = "project",
                                spell_name   = "omen",
                                contributors = \'["hdser"]\') }}'
```